Userland/Utilities/readelf.cpp

```diff
@@ -186,30 +186,6 @@ static char const* object_symbol_binding_to_string(ElfW(Word) type)
 static char const* object_relocation_type_to_string(ElfW(Word) type)
 {
     switch (type) {
-#if ARCH(I386)
-    case R_386_NONE:
-        return "R_386_NONE";
-    case R_386_32:
-        return "R_386_32";
-    case R_386_PC32:
-        return "R_386_PC32";
-    case R_386_GOT32:
-        return "R_386_GOT32";
-    case R_386_PLT32:
-        return "R_386_PLT32";
-    case R_386_COPY:
-        return "R_386_COPY";
-    case R_386_GLOB_DAT:
-        return "R_386_GLOB_DAT";
-    case R_386_JMP_SLOT:
-        return "R_386_JMP_SLOT";
-    case R_386_RELATIVE:
-        return "R_386_RELATIVE";
-    case R_386_TLS_TPOFF:
-        return "R_386_TLS_TPOFF";
-    case R_386_TLS_TPOFF32:
-        return "R_386_TLS_TPOFF32";
-#else
```

Comment:
this should be converted to an if ARCH(x86_64) check, we will need to add aarch64 hooks here eventually